cypress/e2e/functional/opal/manualAccountCreation/welshLanguages/PO-465_languagePreferencesOnAccountDetails.feature

```diff
@@ -23,6 +23,9 @@ Feature: PO-465 language preferences page for all defendant types
 
     Then I see "Document language" has a change link in the account details table
     And I see "Hearing language" has a change link in the account details table
+
+
+
```

Comment:
Is there supposed to be a check in here?

Reply:
deleted few steps